README.md

```diff
@@ -5,7 +5,7 @@
 [![Coveralls][coveralls-badge]][coveralls]
 
 [build-badge]: https://img.shields.io/travis/user/repo/master.png?style=flat-square
-[build]: https://travis-ci.org/user/repo
+[build]: https://travis-ci.org/aw-yt/drawit
```

Comment:
@umarashfaq This badge wasn't working before however we would need to change the URL once you authenticate travis-ci with it